simulation/__main__.py

```diff
@@ -65,12 +68,28 @@ def restrict_between_0_and_1(input_value: float):
         help="A factor by which current (2021) air source heat pump unit+install costs will have declined by, as of the end of 2022",
     )
 
+    def is_isoformat_datetime_string(string):
+        datetime.datetime.fromisoformat(string)
+        return string
+
+    parser.add_argument(
+        "--seed",
+        default=datetime.datetime.now().isoformat(),
+        type=is_isoformat_datetime_string,
+        help="""
+        Seed for random number generator. Default is now.
+    """,
+        metavar="YYYY-MM-DD[*HH[:MM[:SS[.fff[fff]]]]",
+    )
+
     return parser.parse_args(args)
 
 
 if __name__ == "__main__":
     args = parse_args()
 
+    random.seed(args.seed)
```

Comment:
I think it's good to have a test for `PYTHONHASHSEED='0'` but to enforce it we could have a `setup()` function or something in the `if __name__ == "__main__":` here like:
```
import os
import sys
hashseed = os.getenv('PYTHONHASHSEED')
if not hashseed:
    os.environ['PYTHONHASHSEED'] = '0'
    os.execv(sys.executable, [sys.executable] + sys.argv)
```
source: https://stackoverflow.com/a/64138050

Will leave that thought with you, feel free to resolve if you don't think its useful / too complex.

Reply:
If we are going to stick with the current approach (and not do the above, which I am fine with) could we add a `.env.example` with `PYTHONHASHSEED='0'` to the repo and add an instruction to the `README.md` to `cp .env.example .env` before running the code? Thanks.

Reply:
Yeah, I think it's too complicated.

I'll add it to `.env.template`.